src/main/java/prefs/Preferences.java

```diff
@@ -15,23 +15,30 @@
  * It cannot be as we need to mock it.
  */
 public class Preferences { // NOPMD
+
     public static final String DIRECTORY = "settings";
 
     // Standard config filenames used for application and testing
-    public static final String GLOBAL_CONFIG_FILE = "global.json";
-    public static final String TEST_CONFIG_FILE = "test.json";
+    public static final String SESSION_CONFIG_FILENAME = "global.json";
+    public static final String TEST_SESSION_CONFIG_FILENAME = "test.json";
 
-    private final ConfigFileHandler fileHandler;
+    private final ConfigFile sessionConfigFile;
 
-    public GlobalConfig global;
+    private SessionConfig sessionConfig;
 
+    /**
+     * Private constructor that prevents external instantiation
```

Comment:
I think he just added the comment. I am not sure what was the original intention of making the constructor private, but I assume it is to prevent creating `Preferences` manually, and instead use `.create()` or `.load()`